config/samples/sources/azureeventhubsource.yaml

```diff
@@ -19,10 +19,9 @@ kind: AzureEventHubSource
 metadata:
   name: sample
 spec:
-  hubName: insights-activity-logs
-  hubNamespace: demo-event-source
+  eventHubID: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/MyGroup/providers/Microsoft.EventHub/namespaces/MyNamespace/eventHubs/MyEvents
```

Comment:
This line is this PR in a nutshell.

No more conditional requirement of `hubName` and `hubNamespace`. The user sets the resource ID of the Event Hubs instance, and we do the rest:
- validate using OpenAPI, not a webhook
- figure out the namespace + name
- set the resource ID as CloudEvent 'source' attribute